setup.py

```diff
@@ -66,7 +67,7 @@
 install_requires = [
     'bravado>=9.0.6',
     'click>=6.7',
-    'cwltool',
+    'cwltool==1.0.20180326152342',
```

Comment:
@dinosk why this? wasn't CWL test suite passing already?

Reply:
@diegodelemos requirements-builder requires a specified version